ios/PresentationLayer/Recipes/Sources/Recipes/RecipesFlowController.swift

```diff
@@ -11,15 +11,34 @@ enum RecipesFlow: Flow, Equatable {
     case recipes(Recipes)
     
     enum Recipes: Equatable {
+        static func == (lhs: RecipesFlow.Recipes, rhs: RecipesFlow.Recipes) -> Bool {
+            switch (lhs, rhs) {
+            case (.showCounter, .showCounter): true
+            case (.showBooks, .showBooks): true
+            case (.showSkeleton, .showSkeleton): true
+            case (.showImages, .showImages): true
+            case (.showMaps, .showMaps): true
+            case (.showMediaPicker, .showMediaPicker): true
+            case (.presentPickerModal, .presentPickerModal): true
+            default: false
+            }
+        }
+        
         case showCounter
         case showBooks
-        case showRocketLaunches
+//        case showRocketLaunches
         case showSkeleton
         case showImages
         case showMaps
+        case showMediaPicker
+        case presentPickerModal(delegate: MediaPickerSource)
     }
 }
 
+public protocol MediaPickerSource: AnyObject {
+    var media: Binding<[MediaType]> { get }
+}
```

Comment:
Isn't this a general protocol that can be used in any package? If so, it should be in `UIToolkit` instead.